src/testing/tokenizer.rs

```diff
@@ -387,3 +358,143 @@ pub fn fixtures() -> impl Iterator<Item = FixtureFile> {
         fixture_from_path(&path).ok()
     })
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
In addition to simplifying the names of the tokens, these unit tests, and the deserialization needed for them to work, are one of the main purposes of this PR.